src/latbin/lattice-to-fst.cc

```diff
@@ -44,9 +89,28 @@ int main(int argc, char *argv[]) {
         " e.g.: lattice-to-fst  ark:1.lats ark:1.fsts\n";
       
     ParseOptions po(usage);
+    po.Register("read-compact", &read_compact, 
+                "Read compact lattice. Make this false to convert a "
+                "non-compact lattice to an FST. --project-input or --project-output "
+                "can be used to get FSA transition-ids or word labels as labels."
+                "--convert-to-pdf-labels can be used to convert transition-ids to pdf-id+1");
     po.Register("acoustic-scale", &acoustic_scale, "Scaling factor for acoustic likelihoods");
     po.Register("lm-scale", &lm_scale, "Scaling factor for graph/lm costs");
     po.Register("rm-eps", &rm_eps, "Remove epsilons in resulting FSTs (in lazy way; may not remove all)");
+    po.Register("convert-to-pdf-labels", &convert_to_pdf_labels,
+                "Convert lattice to FST with pdf (1-indexed i.e. pdf_id+1) labels "
+                "at the input side; "
+                "applicable only when --read-compact=false. "
+                "Also supply --project-input to get FSA of pdf_id+1 labels.");
+    po.Register("trans-model", &trans_model,
+                "Transition model. This is only required if "
+                "--convert-to-pdf-labels is true");
+    po.Register("project-input", &project_input,
+                "Project to input labels (transition-ids); applicable only "
+                "when --read-compact=false");
+    po.Register("project-output", &project_output,
```

Comment:
I think you mean word-ids.  But I don't think this option makes sense.  The lattice would be mostly epsilons.  Can you remove it if it's not necessary.
Is project-input needed?   Please simplify to only what you need for this PR.